File: WPF/WPF_SemesterProject/WPF_SemesterProject/Ship.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Media.Media3D;

namespace WPF_SemesterProject
{
    public class Ship
    {
        public string id { get; set; }
        public string model { get; set; }
        public string starship_class { get; set; }
        public string manufacturer { get; set; }
        public string cost_in_credits { get; set; }
        public string length { get; set; }
        public string crew { get; set; }
        public string passengers { get; set; }
        public string max_atmosphering_speed { get; set; }
        public string hyperdrive_rating { get; set; }
        public string MGLT { get; set; }
        public string cargo_capacity { get; set; }
        public string consumables { get; set; }
        public string name { get; set; }

        public override string ToString()
        {
            return "name: " + name + "\nmodel: " + model + "\nclass: " + starship_class + "\nmanufacturer: " + manufacturer + "\ncosts: " + cost_in_credits + "\nlength: " + length + "\ncrew: " + crew + "\npassengers: " + passengers + "\nmax speed: " + max_atmosphering_speed + "\nhyperdrive-rating: " + hyperdrive_rating + "\nMGLT: " + MGLT + "\ncargo-capacity: " + cargo_capacity + "\nconsumeables: " + consumables;
        }
    }
}

File: Server/syp.shipwars.semesterproject/src/main/java/com/example/syp/shipwars/semesterproject/Ship.java
package com.example.syp.shipwars.semesterproject;

import jakarta.persistence.Id;
import lombok.Getter;
import lombok.Setter;
import org.springframework.data.mongodb.core.mapping.Document;

@Document(collection = "ships")
@Getter
@Setter
public class Ship {
    @Id private String id;
    private String model;
    private String starship_class;
    private String manufacturer;
    private String cost_in_credits;
    private String length;
    private String crew;
    private String passengers;
    private String max_atmosphering_speed;
    private String hyperdrive_rating;
    private String MGLT;
    private String cargo_capacity;
    private String consumables;
    private String name;
    private int sold = 0;
}
